src/pudl/transform/eia861.py

```diff
@@ -34,6 +62,237 @@
 
 logger = logging.getLogger(__name__)
 
+BA_ID_NAME_FIXES = (
```

Comment:
It certainly could be -- I've been thinking that we need some more general / organized infrastructure about how we apply fixes -- which would include storing them as CSVs somewhere (`package_data` I suppose) and having some functions / objects that know how to apply them appropriately, since we need to do kind of a lot of it. I was reluctant to special-case it (even though this is clearly a little ridiculous) without talking/thinking about how we would want to arrange it for fixes in general.

Reply:
that is a good point! do you want to make a more general issue for this and we can address it later?

Reply:
Okay, see #679. Maybe this is something to add to the list of design issues to discuss.  Ugh, I feel like there's more to do than we can do.